greenkeeper.json

```diff
@@ -35,6 +36,5 @@
         "sandbox/example/package.json"
       ]
     }
-  },
-  "ignore": ["@types/node"]
```

Comment:
Why are we removing this ignore line here?

Reply:
It got removed automatically for some reason, but I added it back in a later commit.